src/modules/uxrce_dds_client/dds_topics.yaml

```
@@ -37,6 +37,7 @@ publications:
 
   - topic: /fmu/out/sensor_combined
     type: px4_msgs::msg::SensorCombined
+    polling_interval: 1
```

Add a minimum this needs units, although I also wonder if phrasing it as maximum rate (Hz) would be more intuitive for most people? I think it's also important to mention it's a max, not necessarily the actual rate you're going to get.

```suggestion
    max_rate_hz: 1000
```

---

That makes sense.

I prefer to use the interval as it is closer to what actually happens (polling). But I understand a `hz` number is more intuitive. I will update that :)

---

That makes sense.

I prefer to use the interval as it is closer to what actually happens (polling). But I understand a hz number is more intuitive. I will update that.

---

In the case of uORB that interval is the minimum time between reported updates.